task01-html-css/task01.css

```diff
@@ -0,0 +1,55 @@
+/**
+* All Rights Reserved
+* This software is property information of AKUREY S.A.
+* Contact us at contact@akurey.com
+* @summary task 01 style sheet
+*/
+.grid-container{
```

Comment:
Really nice approach overall, so because of that I'm gonna get a little picky :)
For more clarity on the code, let's leave an empty space, like this:
`.grid-container {`
I will make the code look nicer and easier to read.